src/lib/update-alternatives/model/alternative.rb

```
@@ -95,6 +95,17 @@ def empty?
       false
     end
 
+    def choice(new_choice_path)
+      return unless choices.map(&:path).include?(new_choice_path)
```

silently ignore problem? I think exception is better here